di2f/index.md

```diff
@@ -0,0 +1,226 @@
+# Introduction
+
+A major promise of the decentralized web is putting ownership back into the hands of users by shifting publishing, discovery, and services from a few core providers, onto the "edge" and into the hands of users/creators. True networks of untrusted peers that can coordinate [@barabas2017defending] to provide important peer-to-peer (p2p) infrastructure and services.
+
+Core components of the decentralized (or distributed) web, such as storage, have received a great deal of attention, including projects such as the Interplanetary File System (IPFS) [@benetIPFSContentAddressed2014], Filecoin [@protocollabsFilecoinDecentralizedStorage2017], Sia [@vorick2014sia], and Storj [@Wilkinson14storja]. These projects are demonstrating that distributed, (incentivized) storage is possible, and even profitable. However, _services_ such as those traditionally found in centralized systems via open (or closed) application programing interfaces (APIs) are less common in p2p systems.
+
+The "Thread Network" is a a proof-of-concept network that aims to address this shortcoming. The proposed system consists of a network of peers working to manage the discovery and coordination of an unbounded set of p2p services. In this paper, we focus on a core piece of this proposed system: service discovery. We propose the use of decentralized identifiers (DID) as a means to coordinate peer-to-peer services.
+
+# Background
+
+## Decentralized Identifiers
+
+A decetralized identifier, or DID, is a string identifier of a _subject_, controlled by a _controller_. A DID might be used to encode a reference to an Ethereum address [@ethrDidResolver] on the Ethereum [buterin2014next,@wood2014ethereum] Mainnet, identify a resource on a network of IoT (Internet of Things) devices, or even represent a unique "identity" such as a user or organization [@ceramic2021Protocol]. For DIDs to be useful, they must be "resolvable" without reliance on a centralized network component. [@Fig:did-overview] shows the interactions between DID components.
+
+![The basic components of DID architecture. Source: [@reed2020decentralized] sec 1.3](./assets/did-overview.png){#fig:did-overview}
+
+The _Verifiable Data Registry_ in @fig:did-overview could be a blockchain or peer-to-peer network. The meaning of _verification_ differs between DID implementations -- some provide a high level of verification through blockchain transactions, while others rely purely on the assumption that the majority of peers are _good actors,_ e.g., IPID (an IPNS-based DID) or Ceramic.
+
+<!-- @sanderpick is this true re: Ceramic? I'm not sure I follow this statement fully? -->
```

Comment:
@sanderpick is this true re: Ceramic? I'm not sure I follow this statement fully?

Reply:
Yeah, not true re ceramic... didn't realize then. Can remove.